proof/refine/ARM/Ipc_R.thy

```diff
@@ -1535,11 +1535,11 @@ lemma makeFaultMessage_inv[wp]:
   done
 
 lemmas threadget_fault_corres =
-          threadget_corres [where r = fault_rel_optionation
+          threadGet_corres [where r = fault_rel_optionation
```

Comment:
In the same spirit as `threadget_corres` -> `threadGet_corres`, this one would be `threadGet_tcbFault_corres`, maybe?

Reply:
Good idea, wasn't included because I was only searching for the `corres:` pattern

Reply:
Ok, it sounds like you could do another pass to see if there are more cases that you couldn’t catch with the `corres:` pattern?